prisma/seed.ts

```diff
@@ -28,11 +28,27 @@ async function main() {
     where: { slug: testFamily.slug },
   });
 
+  // eslint-disable-next-line fp/no-let
+  let family: Family;
   if (!isTestFamilyExists) {
-    await prisma.family.create({ data: testFamily });
+    family = await prisma.family.create({ data: testFamily });
     console.log(
       `test family created, test family name is "${testFamily.name}"`,
     );
+  } else {
+    family = isTestFamilyExists;
+  }
+
+  const testHouseholder = { name: faker.name.fullName() };
+  const isTestHouseholderExits = await prisma.houseHolder.findFirst();
+
+  if (!isTestHouseholderExits) {
+    await prisma.houseHolder.create({
+      data: { ...testHouseholder, familyId: family.id },
```

Comment:
you didn't use testHouseholder variable outside of this if, so it's better to put this inside it.